index.html

```diff
@@ -270,87 +329,289 @@
+
+It is important to keep in mind that there are <em>always</em> privacy [=rules=]. There is no
+purportedly neutral situation in which no [=rules=] apply. We must therefore determine which [=rules=]
+best align with ethical Web values ([[?ETHICAL-WEB]], [[?Why-Privacy]]).
+
+<dfn data-lt='information'>Information flows</dfn> as understood in this document are information
+exchanged or processed by [=actors=]. The information itself need not necessarily be
+[=personal data=]. Disruptive or interruptive information flowing <em>to</em> a
+person is in scope, as is [=de-identified=] [=data=] that can be used to manipulate people or that
+was extracted by observing people's behaviour on someone else's website.
+
+[=Information flows=] need to be understood in a bidirectional manner: the flow of information
+<em>about</em> a person (the subject) just as well as the flow of information <em>towards</em> a
+person (the recipient). Recipients can have their privacy violated in multiple ways such as
+unexpected shocking images, loud noises while one intends to sleep, manipulative information,
+interruptive messages when a person's focus is on something else, or harassment when they seek
+social interactions.
+
+## Individual Autonomy {#autonomy}
 
-<dfn>Privacy</dfn> is an essential value of the Web ([[?ETHICAL-WEB]],
-[[?design-principles]]).
+A [=person=]'s <dfn data-lt="autonomous">autonomy</dfn> is their ability to make decisions of their own volition,
+without undue influence from other parties. People have limited intellectual resources and
+time with which to weigh decisions, and by necessity rely on shortcuts when making
+decisions. This makes their privacy preferences malleable and susceptible to
+manipulation ([[?Privacy-Behavior]], [[?Digital-Market-Manipulation]]). A [=person=]'s [=autonomy=] is enhanced by a
+system or device when that system offers a shortcut that aligns more with what that [=person=] would
+have decided given arbitrary amounts of time and relatively unlimited intellectual ability;
+and [=autonomy=] is decreased when a similar shortcut goes against decisions made under such
+ideal conditions.
 
-<aside class="issue">
+Affordances and interactions that decrease [=autonomy=] are known as <dfn>dark patterns</dfn>.
+A [=dark pattern=] does not have to be intentional, the deceptive effect is sufficient to
+define them ([[?Dark-Patterns]], [[?Dark-Pattern-Dark]]).
 
-Improve this introduction to introduce the several forms of privacy covered in this
-document. The term privacy covers several related topics, both about having <a
-href="#data-privacy">reasonable control over other people's impressions of you</a>, and <a
-href="#intrusion">preventing them from bothering you</a>. Privacy can be <dfn
-data-lt="privacy violation">violated</dfn>.
+Because we are all subject to motivated reasoning, the design of defaults and affordances
+that may impact [=autonomy=] should be the subject of independent scrutiny.
```

Comment:
I am having trouble grasping the breadth of the notion of autonomy so I wanted to suggest an example to discuss.

Consider an engineer who contributes to a W3C draft using their real-world identity, as seems to be customary in the W3C. Their motivation is to create value and to build a personal reputation. ​How does this use of real-world identity increase or decrease autonomy? How are the benefits of participating, accrual of reputation and creation of value for society, recognized by the model of autonomy?

Reply:
I'm not sure what exactly you are looking for. Editors have the option of contributing to W3C work under whatever identity they prefer and I don't believe that there is any expectation that they would have to use one of their legal identities. We have contributors who use pseudonyms, we would not deadname someone just because they haven't obtained a legal change of name, etc.

I don't understand the idea of reputation in the context of the whole of society? I can easily think of cases in which editing a spec is a reputational plus in one context and a minus in another.

Autonomy is enhanced to the extent that people can present their chosen identity and not have their reputation interfered with between different circles, as usual balanced against collective considerations of safety.